core/controlplane/config/templates/cluster.yaml

```diff
@@ -239,7 +239,7 @@ worker:
 #      # If omitted, public subnets are created by kube-aws and used for worker nodes
 #      subnets:
 #      - # References subnets defined under the top-level `subnets` key by their names
```

Comment:
@sathiyas If that's the case, I'd suggest moving the yaml comment outside of the array item:

```yaml
#      subnets:
#      # References subnets defined under the top-level `subnets` key by their names
#      - name: ManagedPublicSubnet1
```

so that one never fails like that.
What do you think?